### PR TITLE
Sources modal: Multiple indicators

### PR DESCRIPTION
**Pull request by sophiamersmann**
Displays expandable tabs for multiple indicators.

I haven't been very disciplined about pushing changes to the branches they belong to. So, this branch also contains various bug fixes that touch on data pages and the sources modal.

### CODE REVIEW

**Comment by sophiamersmann** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2794** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2794" target="_blank">Graphite</a> 
    * **PR #2853** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2853" target="_blank">Graphite</a> 
      * **PR #2877** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2877" target="_blank">Graphite</a> 
        * **PR #2886** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2886" target="_blank">Graphite</a>  👈
    * **PR #2795** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2795" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2886?utm_source=stack-comment).

File: packages/@ourworldindata/core-table/src/CoreTableColumns.ts [hunk 1]
```diff
@@ -258,11 +258,11 @@ export abstract class AbstractCoreColumn<JS_TYPE extends PrimitiveType> {
     }
 
     @imemo get name(): string {
-        return this.def.name ?? this.def.slug
+        return this.def.name || this.def.slug
     }
 
     @imemo get displayName(): string {
-        return this.display?.name ?? this.name ?? ""
+        return this.display?.name || this.name || ""
```
Comment (sophiamersmann) [inline packages/@ourworldindata/core-table/src/CoreTableColumns.ts:265]: this is why the svg tester comes back with a few updated charts

File: packages/@ourworldindata/components/src/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    labels: string[]
+    activeIndex: number
+    setActiveIndex: (label: number) => void
+    horizontalScroll?: boolean
+    slot?: JSX.Element
+}) => {
+    const container = useRef<HTMLDivElement>(null)
+
+    // roving tabindex for keyboard navigation
+    function getNextIndex(eventKey: string): number {
```
Comment (danyx23) [inline packages/@ourworldindata/components/src/Tabs/Tabs.tsx:20]: ❤️

**Review by danyx23 — CHANGES_REQUESTED** [review]
Looks good! I have a few remarks about the wording in slack and I think we should only show multiple indicator tabs if there is a difference between the content we show for each one (i.e. if all are the same we don't show any tabs; if there are differences it is fine to show all indicators, even if most are the same - i.e. no need to de-duplicate further; but all sources being the same is a common case due to our need to split multi-dimensional indicators and we should not confuse our users with this if we don't have to)

**Pull request merged**